ui-lib/src/main/java/cash/z/ecc/ui/MainActivity.kt

```diff
@@ -41,9 +43,19 @@ class MainActivity : ComponentActivity() {
 
     @Composable
     private fun WrapBackup(persistableWallet: PersistableWallet) {
-        BackupWallet(persistableWallet, backupViewModel.backupState, backupViewModel.testChoices) {
+        BackupWallet(
+            persistableWallet, backupViewModel.backupState, backupViewModel.testChoices,
+            onCopyToClipboard = {
+                val clipboardManager = getSystemService(ClipboardManager::class.java)
```

Comment:
I want to point out this code, which is probably what the ticket refers to as "build a reusable logic". 

There are many use cases where there's an action presented to the user to copy a string to clipboard. So what we would want here is to have some abstraction/helper that avoids repeating code when there's a button to copy some string present in a ViewModel.